[importer] Extend README.md (#1966)
kubernetes-sigs/kueue: cmd/importer/README.md

- The rules are evaluated in order.
- `skip: true` can be used to ignore the pods matching a rule.

#### Other flags

> After building the executable the full list of supported flags can be retrieve by:

Reviewer: ```suggestion
After building the executable, the full list of supported flags can be retrieved by:
```

Author: done

  -c, --concurrent-workers uint       number of concurrent import workers (default 8)
      --dry-run                       don't import, check the config only (default true)
  -h, --help                          help for import
  -n, --namespace strings             target namespaces (at least one should be provided)
      --qps float32                   client QPS, as described in https://kubernetes.io/docs/reference/config-api/apiserver-eventratelimit.v1alpha1/#eventratelimit-admission-k8s-io-v1alpha1-Limit (default 50)
      --queuelabel string             label used to identify the target local queue
      --queuemapping stringToString   mapping from "queuelabel" label values to local queue names (default [])
      --queuemapping-file string      yaml file containing extra mappings from "queuelabel" label values to local queue names

Global Flags:
  -v, --verbose count   verbosity (specify multiple times to increase the log level)

```

- At least one `namespace` needs to be specified
- `queuelabel` and `queuemapping` should always be used together.
- One and only one of `queuemapping` and `queuemapping-file` should always be provided.

### Import

> After which, if `--dry-run=false` was specified, for each selected Pod the importer will:

Reviewer: ```suggestion
When running the importer, if `--dry-run=false` was specified, for each selected Pod the importer will:
```

Author: done
